From: Markus Koschany <apo@debian.org>
Date: Tue, 5 Jul 2016 18:50:20 +0200
Subject: gcc-6 literal suffix

Fix GCC-6 error: C++11 requires a space between literal and string macro

Debian-Bug: https://bugs.debian.org/811696
---
 src/config/Functions.cc | 4 ++--
 src/config/Run.cc       | 2 +-
 src/core/Main.cc        | 2 +-
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/config/Functions.cc b/src/config/Functions.cc
index 36ad29d..7229581 100644
--- a/src/config/Functions.cc
+++ b/src/config/Functions.cc
@@ -11,7 +11,7 @@
 #include "Run.h"
 #include "OptParser.h"
 
-#define ICON_DEFAULT PIXMAPDIR"/"PACKAGE_NAME".png"
+#define ICON_DEFAULT PIXMAPDIR "/" PACKAGE_NAME ".png"
 
 static const gchar *authors[] =
 {
@@ -304,7 +304,7 @@ void set_config_states(std::string command)
         
         if (command.empty())
         {
-            command = PACKAGE_NAME" "DEFAULT_ARGV;
+            command = PACKAGE_NAME " " DEFAULT_ARGV;
         }
 
         if (argc <= 1 || tmpoptparser.isSet( OptParser::CONFIG ))
diff --git a/src/config/Run.cc b/src/config/Run.cc
index 229d01d..7c853bd 100644
--- a/src/config/Run.cc
+++ b/src/config/Run.cc
@@ -136,7 +136,7 @@ void Run::stop()
 {
     if (Run::getPID() > 0)
     {
-        if (system ("killall "PACKAGE_NAME) != 0)
+        if (system (" killall " PACKAGE_NAME) != 0)
         {
             std::cout << _("Error kill program: ") << PACKAGE_NAME << std::endl;
         }
diff --git a/src/core/Main.cc b/src/core/Main.cc
index 2ee40f4..c170e63 100644
--- a/src/core/Main.cc
+++ b/src/core/Main.cc
@@ -81,7 +81,7 @@ int main(int argc, char **argv)
 
         if (command.empty())
         {
-            command = PACKAGE_NAME" "DEFAULT_ARGV;
+            command = PACKAGE_NAME " " DEFAULT_ARGV;
         }
 
         if (argc <= 1 || tmpoptparser.isSet( OptParser::CONFIG ))
